<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui::IdentityDialog</class>
 <widget class="QDialog" name="Gui::IdentityDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="realNameLabel">
       <property name="text">
        <string>Real name:</string>
       </property>
       <property name="buddy">
        <cstring>realNameLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="realNameLineEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="userNameLabel">
       <property name="text">
        <string>User name:</string>
       </property>
       <property name="buddy">
        <cstring>userNameLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="userNameLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="nicknamesGroupBox">
     <property name="title">
      <string>Nicknames</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QListWidget" name="nicknamesListWidget">
        <property name="editTriggers">
         <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
        </property>
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::InternalMove</enum>
        </property>
        <property name="defaultDropAction">
         <enum>Qt::MoveAction</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QToolButton" name="addToolButton">
          <property name="text">
           <string>+</string>
          </property>
          <property name="icon">
           <iconset theme="list-add" resource="resources.qrc">
            <normaloff>:/icons/resources/icons/list-add.png</normaloff>:/icons/resources/icons/list-add.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="deleteToolButton">
          <property name="text">
           <string>-</string>
          </property>
          <property name="icon">
           <iconset theme="list-remove" resource="resources.qrc">
            <normaloff>:/icons/resources/icons/list-remove.png</normaloff>:/icons/resources/icons/list-remove.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="upToolButton">
          <property name="text">
           <string>▲</string>
          </property>
          <property name="icon">
           <iconset theme="go-up" resource="resources.qrc">
            <normaloff>:/icons/resources/icons/go-up.png</normaloff>:/icons/resources/icons/go-up.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="downToolButton">
          <property name="text">
           <string>▼</string>
          </property>
          <property name="icon">
           <iconset theme="go-down" resource="resources.qrc">
            <normaloff>:/icons/resources/icons/go-down.png</normaloff>:/icons/resources/icons/go-down.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>userNameLineEdit</tabstop>
  <tabstop>realNameLineEdit</tabstop>
  <tabstop>nicknamesListWidget</tabstop>
  <tabstop>addToolButton</tabstop>
  <tabstop>upToolButton</tabstop>
  <tabstop>downToolButton</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>IdentityDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>IdentityDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
